src/components/TableDialog.vue

```diff
@@ -295,9 +301,11 @@ export default defineComponent({
         await api.uploadTable(workspace, fileName.value, {
           type: 'csv',
           data: selectedFile.value,
+          edgeTable: edgeTable.value,
           key: keyField.value,
           overwrite: overwrite.value,
           columnTypes: columnType.value,
+          url: host,
```

Comment:
Same as roni's comment about the corresponding change in multinetJS

Reply:
See 5f46644